clients/Application/package.json

```diff
@@ -1,73 +1,48 @@
 {
-  "name": "application",
+  "name": "dashboard",
```

Comment:
Both Admin and Application client projects have name as "dashboard" is that correct?